app/views/partials/_booking.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="container">
```

Comment:
@devc0der20 - why isn't this partial in the views/bookings/ directory since it's a booking partial?

Reply:
don't mind the partials you can delete them I was just testing without affecting the code of the others!

Reply:
You need to change that and push again. Thx